provisioning/baremetal_pod.go

```diff
@@ -700,8 +699,7 @@ func createContainerMetal3RamdiskLogs(images *Images) corev1.Container {
 		},
 		TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
 		SecurityContext: &corev1.SecurityContext{
-			RunAsUser:  ptr.To(ironicUserID),
-			RunAsGroup: ptr.To(ironicGroupID),
+			Privileged: pointer.BoolPtr(true),
```

Comment:
Have you tried adding CAP_FOWNER instead?
As far as I know, privileged mode should only be needed to tell SELinux to [allow us to use hostpath volumes](https://docs.openshift.com/container-platform/4.15/storage/persistent_storage/persistent_storage_local/persistent-storage-hostpath.html#persistent-storage-hostpath-pod_persistent-storage-hostpath), (which the shared volume is not an example of). In theory I believe everything else should be controllable by capabilities.

Reply:
All for trying different things, but let's limit the experiments to a reasonable timeframe since this is a very high priority issue.

Reply:
And if so, shouldn't it be `CAP_DAC_OVERRIDE` instead?

Reply:
It appears CAP_DAC_OVERRIDE should cover it, and CAP_FOWNER is a level above that, which is not needed.